library/core/src/commonMain/kotlin/com/attafitamim/krop/core/crop/CropState.kt

```diff
@@ -74,9 +74,9 @@ fun cropState(
         _region = new.asMatrix(src.size).map(unTransform.map(region))
     }
 
-    override fun reset() {
+    override fun reset(forceShape: CropShape?) {
```

Comment:
Not 100% satisfied with how I implemented this, but I could not easily find another way...

Reply:
As a follow-up to my other comment about lists, can we pass an optional `CropperStyle` to the `reset` method instead of `CropShape`? And then use the same mechanism to determine the `defaultShape`?